query-connector/src/app/query/page.tsx

```diff
@@ -40,7 +43,11 @@ const Query: React.FC = () => {
   return (
     <>
       <SiteAlert page={mode} />
+      {Object.keys(CUSTOMIZE_QUERY_STEPS).includes(mode) && (
```

Comment:
Think there might be some extra bottom spacing here between the main content container. Think Figma only has 1.5 rem of margin between the step counter and the start of the nav links while the implementation has 3 rem of padding and then some in the main container.

<img width="657" alt="Screenshot 2024-10-17 at 8 50 51 AM" src="https://github.com/user-attachments/assets/1574e515-aac0-4844-9035-172f120447be">

<img width="1301" alt="Screenshot 2024-10-17 at 8 51 20 AM" src="https://github.com/user-attachments/assets/b406b3e2-77ea-4569-b1ea-fff9817ae011">